client/ayon_houdini/plugins/publish/collect_componentbuilder_lop.py

```diff
@@ -34,14 +34,18 @@ def process(self, instance):
         # TODO: Do we want this? or use existing frames? Usually a Collector
         #  should not 'extract' but in this case we need the resulting USD
         #  file.
-        node.cook(force=True)  # required to clear existing errors
-        node.parm("execute").pressButton()
-
-        errors = node.errors()
-        if errors:
-            for error in errors:
-                self.log.error(error)
-            raise PublishError(f"Failed to save to disk '{node.path()}'")
+        try:
+            node.cook(force=True)  # required to clear existing errors
+            node.parm("execute").pressButton()
+        except Exception as exc:
```

Comment:
Why is this needed?

Reply:
When it fails to cook, it throws this error: 
(it happens a little before `raise PublishError`)
```python
Traceback (most recent call last):
  File "C:\Users\Mustafa Taher\AppData\Local\Ynput\AYON\dependency_packages\ayon_2411271645_windows.zip\dependencies\pyblish\plugin.py", line 528, in __explicit_process
    runner(*args)
  File "E:\Ynput\ayon-houdini\client\ayon_houdini\plugins\publish\collect_componentbuilder_lop.py", line 38, in process
    node.cook(force=True)  # required to clear existing errors
    ^^^^^^^^^^^^^^^^^^^^^
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.370/houdini/python3.11libs\hou.py", line 17842, in cook
    return _hou.OpNode_cook(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
hou.OperationFailed: The attempted operation failed.
Error while cooking.
```

Reply:
Does this work? I vaguely recall `hou.Error` not inheriting from Python exceptions, hence not getting caught.

Also, I'm not fan of capturing this as globally as this.

Should we be try excepting the `node.cook()` itself, and not the full executing of the parm too - just so we're more explicitly capturing that it's _that_ cook that's failing - and not the render cook.

Technically the user may e.g. currently be on frame 1200 but the ROP set to 1001-1100. A cook on 1200 may hence fail, even though the render could technically exists.

It may be even better if we can improve the code and find a way to 'clear the errors' without forcing a cook of course.

Let's be very explicit about what error we're capturing here and only try..except the `node.cook()` itself.